build.gradle

```diff
@@ -184,6 +191,7 @@ configure(subprojects.findAll {! it.name.startsWith('spark-lineage') }) {
 
   configurations.all {
     exclude group: "io.netty", module: "netty"
+    exclude group: "log4j", module: "log4j"
```

Comment:
This does seem to do something, what is not covered here is the -core and -api modules, but those are locked at the latest version